utils/types/deployment/deployment.go

```diff
@@ -0,0 +1,41 @@
+package deployment
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rudderlabs/rudder-server/config"
+)
+
+type Type string
+
+const (
+	DedicatedType   Type = "DEDICATED"
+	HostedType      Type = "HOSTED"
+	MultiTenantType Type = "MULTITENANT"
+)
+
+const defaultClusterType = DedicatedType
+
+func GetFromEnv() (Type, error) {
+	t := Type(os.Getenv("DEPLOYMENT_TYPE"))
```

Comment:
Why are we using os.Getenv and not the usual config.GetEnv?

Reply:
make sense, replaced it